### PR TITLE
LIVE-14383 working display of team link

### PR DESCRIPTION
**Pull request by Wozacosta**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Third party integration teams, assuming they provided a link, now get's their team names clickable in the "integrated by" drawer

## Before LLM: 

image

## After LLM: 

image

## LLD

image

image

image

image

### ❓ Context

- **JIRA or GitHub link**: 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by hedi-edelbloute — APPROVED** [review]

**Review by LucasWerey — APPROVED** [review]

File: apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx [hunk 1]
```diff
@@ -19,7 +30,34 @@ const Title = styled(Text)`
...
+        verticalAlign={"bottom"}
+        onClick={() => openURL(teamLink)}
+        justifyContent="center"
+        color={"palette.primary.main"}
+      >
+        {team}
+        <ExternalLinkIconContainer>
+          <IconExternalLink size={12} />
+        </ExternalLinkIconContainer>
+      </Link>{" "}
```
Comment (themooneer) [inline apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx:54]: remove the `{" "}`

File: apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx [hunk 2]
```diff
@@ -19,7 +30,34 @@ const Title = styled(Text)`
...
+};
+
+const TeamLink = ({ team, teamLink }: TeamLinkProps) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <Description>{t("account.subHeader.drawer.descriptionLink.integration")}</Description>{" "}
+      <Link
+        verticalAlign={"bottom"}
+        onClick={() => openURL(teamLink)}
```
Comment (themooneer) [inline apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx:46]: can you move this to a handler please ?

File: apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx [hunk 3]
```diff
@@ -45,9 +83,15 @@ export function AccountSubHeaderDrawer({ isOpen, closeDrawer, family, team }: Pr
         </Box>
         <Box py={2}>
           <Description>
-            {t("account.subHeader.drawer.description3", {
-              team,
-            })}
+            {teamLink ? (
+              <TeamLink team={team} teamLink={teamLink} />
+            ) : (
+              <>
```
Comment (themooneer) [inline apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx:89]: you don't need a fragment here, simply the 
```
t("account.subHeader.drawer.description3", {
                  team,
                })
```

Comment (Wozacosta) [reply apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx]: Done in https://github.com/LedgerHQ/ledger-live/pull/8028/commits/b804236dd4d8d8f24b41a6defeb592fe84d8172b 👍

File: apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx (hunk 2 above)
Comment (Wozacosta) [reply apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx]: Done in https://github.com/LedgerHQ/ledger-live/pull/8028/commits/b804236dd4d8d8f24b41a6defeb592fe84d8172b 👍

File: apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx (hunk 1 above)
Comment (Wozacosta) [reply apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx:59]: It's actually required to not have the link stuck to the surrounding works, not using padding/margin allows the renderer to break the line where needed.

Comment (themooneer) [reply apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx:59]: you can use a none braking space , its more native than this. 
`&nbsp;`

Comment (Wozacosta) [reply apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx:59]: Thanks for the tip!

**Review by hedi-edelbloute — APPROVED** [review]

File: apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx [hunk 4]
```diff
@@ -19,7 +30,37 @@ const Title = styled(Text)`
...
+};
+
+const TeamLink = ({ team, teamLink }: TeamLinkProps) => {
+  const onOpenTeam = useCallback(() => {
+    openURL(teamLink);
+  }, [teamLink]);
+
+  return (
+    <Trans i18nKey="account.subHeader.drawer.description3Link">
+      This integration has been carried out by
```
Comment (themooneer) [inline apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx:46]: this string should be translated no ?

File: apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx [hunk 5]
```diff
@@ -19,7 +30,37 @@ const Title = styled(Text)`
...
+        onClick={onOpenTeam}
+        justifyContent="center"
+        color={"palette.primary.main"}
+      >
+        {team}
+        <ExternalLinkIconContainer>
+          <IconExternalLink size={12} />
+        </ExternalLinkIconContainer>
+      </Link>
+      in collaboration with Ledger
```
Comment (themooneer) [inline apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/AccountSubHeaderDrawer.tsx:58]: same question

File: apps/ledger-live-mobile/src/components/AccountSubHeader/AccountSubHeaderDrawer.tsx [hunk 6]
```diff
@@ -1,16 +1,44 @@
...
+  }, [teamLink]);
+
+  return (
+    <Text textAlign={"center"}>
+      <Trans i18nKey="account.subHeader.drawer.description3Link">
+        {"This integration has been carried out by"}
+        <Flex style={{ transform: "translateY(5px)" }}>
+          <ExternalLink type="color" text={team} onPress={onOpenTeam} />
+        </Flex>
+        {"in collaboration with Ledger"}
```
Comment (themooneer) [inline apps/ledger-live-mobile/src/components/AccountSubHeader/AccountSubHeaderDrawer.tsx:35]: same question here

**Pull request merged**